respec-config.js

```diff
@@ -22,7 +22,18 @@ var respecConfig = {
 		company: "IBM",
 		companyURI: "https://www.ibm.com/",
 		w3cid: 46880
-	}],
+	},{
+		Name: "Chris Loiselle",
```

Comment:
@daniel-montalvo Note that I edited the code and fixed the spelling of Chris Loiselle's name.